routing/payment_session.go

```diff
@@ -31,6 +31,12 @@ const (
 	errEmptyPaySession
 )
 
+var (
+	// DefaultMinShardMaxAmt is the default amount beyond which we won't try
+	// to further split the payment if no route is found.
+	DefaultMinShardMaxAmt = lnwire.NewMSatFromSatoshis(10000)
```

Comment:
where does `Max` come from?

Reply:
you mean the var name? It is the minimum amount that we set as a max for splitting. Sentence added

Reply:
Dropped the min-max thing. Now it is just the minimum shard amt